src/main/resources/changesets/changelog-consent-103.0.xml

```diff
@@ -0,0 +1,28 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="103.0" author="connorlbark">
```

Comment:
Huzzah!  We now have two PRs with the same id.

From Liquibase's [documentation](https://docs.liquibase.com/concepts/changelogs/changelog-formats.html): ...
The id tag is only used as an identifier, it does not direct the order that changes are run and does not have to be an integer.

Should we move to a different id value?  Maybe a ticket? "DUOS-2183" or date and ticket "20221115_DUOS-2183"

Reply:
Usually we just update whichever merges in second; so mine will be 104.0 if yours goes in first.

Reply:
I don't have a strong preference for file naming aside from the nicety of seeing all changelogs sequentially ordered. In terms of ID naming options, I lean towards the date-based ids over others.